unicodetools/src/main/java/org/unicode/props/UnicodeProperty.java

```diff
@@ -265,9 +265,10 @@ public String getVersion() {
     }
 
     public Iterable<String> getValues(int codepoint) {
-        return isMultivalued
-                ? delimiterSplitter.split(getValue(codepoint))
-                : Collections.singleton(getValue(codepoint));
+        String value = getValue(codepoint);
+        return isMultivalued && value != null
+                ? delimiterSplitter.split(value)
```

Comment:
optional: does this return a Collection? if so, upgrade the return type?

Reply:
It returns an Iterable. This is a Google thing. Its documentation says `Java 8 users may prefer {@link #splitToStream} instead.`. Do we prefer that?

Reply:
I think I am at least as confused by the Iterable/Collection/Stream situation as you are by C++ iterators…

Reply:
> I think I am at least as confused by the Iterable/Collection/Stream situation as you are by C++ iterators…

I bet I am still more confused ;-)

Reply:
> It returns an Iterable. This is a Google thing. Its documentation says `Java 8 users may prefer {@link #splitToStream} instead.`. Do we prefer that?

Maybe. For the singleton case, we could then presumably use `Stream.of(value)`

Reply:
we could then call this valueStream()

Reply:
Streams are the preferred form where you normally just want to deal with the contents one at a time. You can always create a iterator from a stream.

Reply:
I’ll do that in a later PR, since it is going to affect other callers.